src/app/pipes/nodedisplay.ts

```diff
@@ -41,6 +41,22 @@ export class NodeTypePluralPipe implements PipeTransform {
 }
 
 
+@Pipe({name: 'nodeInternalUrl'})
+export class NodeInternalUrlPipe implements PipeTransform {
+    transform(node: any): string {
+        const baseUrl = 'https://pipeline.engineering.redhat.com/';
+        const resourceType = node.resource_type.toLowerCase();
+        let identifier = '';
```

Comment:
How about keeping it as undefined with `let identifier;`. Having it assigned to an empty string doesn't have any particular meaning.